R/age_group.R

```diff
@@ -0,0 +1,60 @@
+#' Create age groups
+#'
+#' @description
+#' A wrapper for \code{\link[base]{cut}} tailored for creating age groups. For
+#' each value of integer \code{x}, what age group does it fall into.
+#'
+#' @param x a vector of numeric values
+#' @param from,to,by start, end and increment values for a sequence of age
+#'   groups.
```

Comment:
I think I would do a separate param for each of these and explain what the default values are.